vanilla/vanilla
# Fix embedded images ignoring alignment

Closes vanilla/support#1902

This PR fixes a problem where images embedded in signatures aren't aligning properly with the rich editor. This adds a css condition that respects the alignment chosen in the editor.

### TO TEST
1. Enable rich editor and the signatures plugin and create a signature with an image aligned to the left (or right).
1. Create a post and note that the image is center-aligned.
1. Check out this branch and verify that the image is left- (or right-) aligned.
